pages/textbook-beta/course/quantum-machine-learning-course.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <main class="quantum-machine-learning-course-page">
+    <TextbookBetaContentMenuSection
+      class="quantum-machine-learning-course-page__content-menu"
+    />
+    <AppPageHeaderWithImage :cta="startLearningCTA" :back-link="backToTextbookHomeLink">
+      <template slot="title">
+        {{ headerTitle }}
+      </template>
+      <template slot="description">
+        <p
+          v-for="(paragraph, index) in headerDescription"
```

Comment:
since this `headerDescription` is only 1 paragraph, we don't need to iterate here.

this can be updated to:

`<p v-text="headerDescription" />`